### PR TITLE
Adds results from our specs to CodeClimate

### PR DESCRIPTION
**Pull request by Willianvdv**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,5 +2,10 @@ source 'https://rubygems.org'
 
 gem 'pry'
 
+group :test do
+  gem 'simplecov'
+  gem 'codeclimate-test-reporter', '~> 1.0.0'
```
Comment (esjee) [inline Gemfile]: Is it recommended to set a minimum version like this?

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -3,6 +3,9 @@
 require "support/protected_attributes"
 require "pry"
 
+require "simplecov"
+SimpleCov.start
```
Comment (esjee) [inline spec/spec_helper.rb]: Should this be done locally as well, or only on CI?

File: Gemfile (hunk 1 above)
Comment (Willianvdv) [inline Gemfile]: I've copy pasted this from the tutorial, should I remove the version pin?

File: spec/spec_helper.rb (hunk 2 above)
Comment (Willianvdv) [inline spec/spec_helper.rb]: Locally works, we just don't post the results to code climate.

File: Gemfile (hunk 1 above)
Comment (esjee) [inline Gemfile]: Nah

**Pull request merged**